packages/quick_actions/lib/quick_actions.dart

```diff
@@ -82,8 +83,8 @@ class QuickActions {
   Future<void> clearShortcutItems() =>
       channel.invokeMethod<void>('clearShortcutItems');
 
-  Map<String, String> _serializeItem(ShortcutItem item) {
-    return <String, String>{
+  Map<String, String?> _serializeItem(ShortcutItem item) {
+    return <String, String?>{
```

Comment:
I think ideally we'd make this non-null and omit entries that are null from the dictionary, but that would require changes to the native implementations unfortunately, so LG.